torchvision/datasets/fgvc_aircraft.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+import os
+from typing import Any, Callable, List, Optional, Tuple
+
+import PIL.Image
+
+from .utils import download_and_extract_archive, verify_str_arg
+from .vision import VisionDataset
+
+annotation_level_to_file = {
+    "variant": "variants.txt",
+    "family": "families.txt",
+    "manufacturer": "manufacturers.txt",
+}
+
+
+class FGVCAircraft(VisionDataset):
+    """`FGVC Aircraft <https://www.robots.ox.ac.uk/~vgg/data/fgvc-aircraft/>`_ Dataset.
+
+    The dataset contains 10,200 images of aircraft, with 100 images for each of 102
+    different aircraft model variants, most of which are airplanes.
+    Aircraft models are organized in a four-levels hierarchy. The four levels, from
+    finer to coarser, are:
+
+    - Model, e.g. Boeing 737-76J. Since certain models are nearly visually indistinguishable,
+        this level is not used in the evaluation.
```

Comment:
If we don't expose `annotation_level="model"`, we shouldn't include it in the docstring.